gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1747,10 +1747,12 @@ GbGradeTable.applyStudentFilter = function(data) {
       if (GbGradeTable.settings.isStudentNumberVisible) {
           searchableFields.push(student.studentNumber);
       }
-      var studentSearchString = searchableFields.join(";")
+      var studentSearchString = searchableFields.join(";");
+      studentSearchString = studentSearchString.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
 
       for (var i=0; i<queryStrings.length; i++) {
         var queryString = queryStrings[i];
+        queryString = queryString.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Comment:
Again, just make this a const and a one liner
const queryString = queryStrings[i].normalize("NFD").replace(/[\u0300-\u036f]/g, "");